docs/site/index/process.md

```diff
@@ -24,7 +24,15 @@ The [UTS #35: Locale Data Markup Language (LDML)](https://www.unicode.org/report
 
 - Requests for changes are entered in the bug/feature request database ([file a ticket](/requesting_changes#how-to-file-a-ticket)).
 - Structural changes are always backwards-compatible. That is, previous files will continue to work. Deprecated elements remain, although their usage is strongly discouraged.
-- There is a standing policy for structural changes that require non-trivial code for proper implementation, such as time zone fallback or alias mechanisms. These require design discussions in the Technical Committee that demonstrates correct function according to the proposed specification.
+- There is a standing policy for structural changes that require non-trivial code for proper implementation, such as time zone fallback or alias mechanisms. These require design discussions in the [CLDR Design Working Group](https://cldr.unicode.org/cldr-tc/design-wg) and approval by the [CLDR Technical Committee](https://cldr.unicode.org/cldr-tc). Complex changes may require prototypes that demonstrate correct function according to the proposed specification.
+- New sections may be added to the specification with the status of _Technical Preview_ or _Final Candidate_. That depends on how fleshed-out the new section is, what type of feedback the Technical Committee requires, and whether the feedback period needs to extend across one or more releases. These terms mean the following:
```

Comment:
I would reconsider the expression "fleshed-out" as it may be unfamiliar to international readers, and maybe use "finalized and comprehensive" instead.
I would also simplify "These terms mean the following:" to "This means:"

Reply:
reasonable changes, except I think it is better to change "fleshed-out" to just "comprehensive"